ndcube/ndcube.py

```diff
@@ -268,17 +261,17 @@ def pixel_to_world(self, *quantity_axis_list):
         # The docstring is defined in NDDataBase
 
         quantity_axis_list = quantity_axis_list[::-1]
-        pixel_to_world = self.wcs.pixel_to_world(*quantity_axis_list)
+        pixel_to_world = self.high_level_wcs.pixel_to_world(*quantity_axis_list)
         return pixel_to_world[::-1]
 
     def world_to_pixel(self, *quantity_axis_list):
         # The docstring is defined in NDDataBase
         
         quantity_axis_list = quantity_axis_list[::-1]
-        world_to_pixel = self.wcs.world_to_pixel(*quantity_axis_list)
+        world_to_pixel = self.high_level_wcs.world_to_pixel(*quantity_axis_list)
 
         # Adding the units of the output
-        result = [u.Quantity(world_to_pixel[index], unit=u.pix) for index in range(self.wcs.pixel_n_dim)]
+        result = [u.Quantity(world_to_pixel[index], unit=u.pix) for index in range(self.high_level_wcs.pixel_n_dim)]
```

Comment:
Do these need flipping back into numpy order after this?

Reply:
Yeah, as the output is in wcs ordering.